### ISSUE TITLE
Shift and clicking on an item in the crates will give you the item (in Creative mode)

### ISSUE THREAD

**Issue by farizluqman** [ConnorLinfoot/CratesPlus#650]
### Debug Info
VERSION: Using Paperspigot 1.14.2 build 80

PLUGINS:

AdvancedRegionMarket, AsyncWorldEdit, AuctionHouse, BetterRTP, ChatManager, Citizens, CitizensCMD, CMI, Coins, CoreProtect, CrateReloaded, CratesPlus, DeluxeChat, DeluxeMenus, DeluxeTags, DiceFurniture, DiscordSRV, EpicRename, Essentials, EssentialsGeoIP, EssentialsSpawn, FurnitureLib, GriefPrevention, Harbor, HeadsPlus, HolographicDisplays, iConomy, IllegalStack, InventoryRollback, Item2Chat, Jobs, LuckPerms, MaintenanceSpigot, mcMMO, Minepacks, Multiverse-Core, MysqlEcoBridge, PlaceholderAPI, PlayerPoints, PlayerReport, PlugMan, ProtectionLib, ProtectionStones, ProtocolLib, Quests, QuickShop, spark, SpecializedCrates, SuperVanish, TAB, Towny, TownyChat, TownyGUI, TradeSystem, Vault, Votifier, VotingPlugin, WorldEdit, WorldGuard

### Actions to reproduce (if known):
TO REPRODUCE:
Install the plugin, get to creative mode, right click (PREVIEW) on a crate, shift click on the item. You will see that you are able to get the item.

### Link/paste of stack trace/error (if any)
The console is spitting the same errors
```> [12:33:22 ERROR]: Could not pass event PlayerInteractEvent to CratesPlus v4.5.2
java.lang.NullPointerException: null
at plus.crates.Listeners.PlayerInteract.onPlayerInteract(PlayerInteract.java:46) ~[?:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor396.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.14.2.jar:git-Paper-65]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.2.jar:git-Paper-65]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.2.jar:git-Paper-65]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:536) ~[patched_1.14.2.jar:git-Paper-65]
at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:445) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.PlayerInteractManager.a(PlayerInteractManager.java:441) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1393) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:18) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:127) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:105) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.MinecraftServer.aW(MinecraftServer.java:998) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:991) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:115) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:975) ~[patched_1.14.2.jar:git-Paper-65]
at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:908) ~[patched_1.14.2.jar:git-Paper-65]````

**Comment by MrTrinax** [comment]
This issue is also reproduced 1.14.4.

Exploiting this issue is also possible in Survival. You can also Shift + Click an item while opening the case to receive the item. 
This breaks the plugin and is basically unusable cause of the exploitation.

**Comment by ConnorLinfoot** [comment]
Fixed in upcoming update #793
